common/src/main/java/com/revenuecat/purchases/common/AppConfig.kt

```diff
@@ -15,8 +15,15 @@ class AppConfig(
     val store: Store,
     val dangerousSettings: DangerousSettings = DangerousSettings(autoSyncPurchases = true),
     // Should only be used for tests
-    var forceServerErrors: Boolean = false,
+    private val runningTests: Boolean = false,
```

Comment:
I was trying to calculate this within `AppConfig` itself but was getting problems trying to mock these when running unit tests (which don't run in the integrationTest flavor).